tests/e2e/suites/basic/hello_world.go

```diff
@@ -0,0 +1,149 @@
+package basic
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	// "io"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/giantswarm/apiextensions-application/api/v1alpha1"
+	"github.com/giantswarm/apptest-framework/pkg/state"
+	"github.com/giantswarm/clustertest/pkg/application"
+	"github.com/giantswarm/clustertest/pkg/logger"
+	"github.com/giantswarm/clustertest/pkg/wait"
+
+	networkingv1 "k8s.io/api/networking/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func newHelloWorldApp(ingressHost string) (*application.Application, error) {
+	org := state.GetCluster().Organization
+	helloWorldAppValues := map[string]string{"IngressHost": ingressHost}
+	helloWorldApp := application.New(fmt.Sprintf("%s-hello-world", state.GetCluster().Name), "hello-world").
+		WithCatalog("giantswarm").
+		WithOrganization(*org).
+		WithVersion("latest").
+		WithClusterName(state.GetCluster().Name).
+		WithInCluster(false).
+		WithInstallNamespace("default").
+		MustWithValuesFile("./test_data/helloworld_values.yaml", &application.TemplateValues{
```

Comment:
```suggestion
		MustWithValuesFile("./data/hello-world-values.yaml", &application.TemplateValues{
```

Reply:
we use `test_data` in cluster-test-suites already, so I'd sick with it for now